### ISSUE TITLE
请问PartitionMat.txt是用GenMSBTMap生成的吗？那GenMSBTMap.py中pre_path中Label是如何得到的？

### ISSUE THREAD

**Issue by GongchunDing** [AolinFeng/PMP-VVC-TIP2023#3]

**Comment by GongchunDing** [comment]
可以提供一下除RaceHorses外其它序列的partitionMap吗？

**Comment by AolinFeng** [comment]
PartitionMat.txt is used for encoder acceleration and is generated by the "pot_process()" function defined in Metrics.py. The "post_process()" function is called in  Inference_QBD.py.

GenMSBTMap.py is used together with CreateDataset.py to generate the training labels. You can reference the notes at the beginning of the two files. CreateDataset.py will output qt depth map, direction map, and the last layer of bt depth map. GenMSBTMap.py  will use the output of CreateDataset.py to generate the complete partition map (mainly the multi-layer bt depth map ). So, The label of pre_path you asked is the output of CreateDataset.py (qt depth map, direction map, and the last layer of bt depth map). 

In summary, PartitionMat.txt is generated in the inference stage and used for encoder acceleration. GenMSBTMap.py is used in the training label generation stage.

**Comment by AolinFeng** [comment]
> 可以提供一下除RaceHorses外其它序列的partitionMap吗？

Sorry that we cannot provide more examples currently.
